### ISSUE TITLE
Add a test for the CLI's file walk function for the zip creator

### ISSUE THREAD

**Issue by arschles** [gomods/athens#31]
After #30, there will be a tree walk function that we'll need to test

**Comment by marpio** [comment]
@marstr are you on it? If not I could write these tests.

**Comment by karrick** [comment]
I did this for dep a while back. If nobody is working on this, I'd be happy to tackle.

**Comment by karrick** [comment]
Also, working on `dep` I found the standard library `filepath.Walk` function has a bug on Windows. I created this library for it, and it's got some good performance results, including just working correctly on Windows:

https://www.reddit.com/r/golang/comments/83lwfs/a_quick_comparison_between_different_go_file_walk/

**Comment by arschles** [comment]
@karrick happy to have you go give it a shot 😄

**Comment by marstr** [comment]
Howdy folks, sorry I was out for a couple of weeks. I can get on this and have a PR in by end of week. That is, unless @karrick already addressed this.

**Comment by michalpristas** [comment]
nobody addressed it afaik. please check if the work you do does not conflict with #118

**Comment by arschles** [comment]
I am putting this on hold because I am not sure exactly how the CLI is going to look, or whether we are going to need one at all.

**Comment by marstr** [comment]
Sorry, dropping balls over here too!

**Comment by arschles** [comment]
No worries Martin! We all are busy 😊

**Comment by robjloranger** [comment]
Can we close as the CLI is gone? Or will it still be useful elsewhere?

**Comment by michalpristas** [comment]
Yes, indeed.
